README.md

```diff
@@ -1,35 +1,31 @@
-# EIPs [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
-Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.
+# Ethereum Improvement Proposals (EIPs) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-A browsable version of all current and draft EIPs can be found on [the official EIP site](https://eips.ethereum.org/).
```

Comment:
I think this line is useful here. Especially if you want to make the distinction between consuming EIPs (the eips.ethereum.org website) and submitting EIPs (the readme).

Reply:
This line is there but it is moved below the warning about opening new PRs. The motivation of course being that people are opening PRs with reckless abandon whereas nobody is having any trouble finding an EIP if they wanted to.